cypress-tests/cypress/e2e/PaymentUtils/Commons.js

```diff
@@ -637,9 +637,33 @@ export const connectorDetails = {
         status: 200,
         body: {
           status: "requires_payment_method",
+          shipping_cost: 50,
+          amount: 6500,
```

Comment:
Why these have been added here?
`shipping_cost` have not been added in the intent but we're expecting it the response.
Amount is hard coded here which I do not recommend. This can be directly evaluated, right?

Reply:
This is by mistake , will remove